app/src/renderer/components/common/AppHeader.vue

```diff
@@ -8,7 +8,7 @@ nav#app-header: .container
   router-link.header-item.header-item-logo(to="/")
     img(src="~@/assets/images/cosmos.png")
   app-menu(v-if="config.activeMenu === 'app' || config.desktop")
-  app-menu-user(v-if="config.activeMenu === 'user' || config.desktop")
+  // app-menu-user(v-if="config.activeMenu === 'user' || config.desktop")
```

Comment:
comment?

Reply:
We don't have an app-menu-user currently because this code was taken from the cosmos-forum. We may have in the future?